src/utils/libs.ts

```diff
@@ -0,0 +1,12 @@
+export default class Libs {
+  public static validateKey(key: string): string {
+    const err = 'Invalid api key... check https://wakatime.com/settings for your key';
```

Comment:
Let's say `Api key invalid. Find your api key from wakatime.com/settings/api-key`. That's the same message we've been using for the client.